docs/user/getting-started.md

```diff
@@ -47,101 +47,48 @@ NAME  NAMESPACE  REVISION  UPDATED  STATUS  CHART  APP VERSION
 
 ## Deploying the Flight Control Service
 
-### Standalone flightctl with keycloak integration
+OpenShift is the only platform supported for deployment of the Flight Control. This will change in the future.
```

Comment:
```suggestion
```

May be we want to avoid stating that altogether, we don't want to frighten away any community initiative around the project, when all missing is ingress/gateway yamls :)

Reply:
ok, removed :)